.devcontainer/Dockerfile

```diff
@@ -1,3 +1,5 @@
-FROM registry.access.redhat.com/ubi7/go-toolset:1.19
+ARG UBI_VERSION
+
+FROM registry.access.redhat.com/ubi${UBI_VERSION}/go-toolset:1.19
```

Comment:
Did we check if go 1.19 is available on rhel 8 & 9 or do we have to use 1.20 in that case? :)

Reply:
Thanks very much! We checked and they're available for both :)